bin/run-caliper/run-caliper.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+cd ~
+cd hyperledger/caliper-workspace
+npx caliper bind --caliper-bind-sut ethereum --caliper-bind-sdk 1.2.1 --caliper-cwd ./ --caliper-bind-args="-g"
+npx caliper benchmark run --caliper-workspace /hyperledger/caliper-workspace --caliper-benchconfig config.yaml --caliper-networkconfig network.json
```

Comment:
where is netwrok.json

Reply:
after --caliper-networkconfig

Reply:
it should be generated after we run the script. based on the user's input

Reply:
are you sure? @boxrhcp can you advice?

Reply:
I updated the task in case it was not clear enough. This script should receive as parameters some values that are needed to change the network config file. Then it should execute caliper.